spec/presenters/link_api_response_presenter_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe LinkApiResponsePresenter do
+  describe '#present' do
+    let(:authority) { FactoryGirl.build(:local_authority) }
+    let(:presenter) { described_class.new(authority, link) }
```

Comment:
It feels a little surprising to specify a thing without all its dependencies (`link`) being defined.  Is this a pattern I should be using?